README.md

```diff
@@ -288,16 +288,16 @@ structure, which defines a *Redis object*:
 Basically this structure can represent all the basic Redis data types like
 strings, lists, sets, sorted sets and so forth. The interesting thing is that
 it has a `type` field, so that it is possible to know what type a given
-object is, and a `refcount`, so that the same object can be referenced
+object has, and a `refcount`, so that the same object can be referenced
```

Comment:
I think "is" was the correct one here as the type field is identifying the type of the object, making it an "is a" rather than a "has a" relationship.